crates/bevy_solari/src/realtime/restir_gi.wgsl

```diff
@@ -25,7 +25,8 @@ struct PushConstants { frame_index: u32, reset: u32 }
 var<push_constant> constants: PushConstants;
 
 const SPATIAL_REUSE_RADIUS_PIXELS = 30.0;
-const CONFIDENCE_WEIGHT_CAP = 30.0;
+const CONFIDENCE_WEIGHT_CAP = 8.0;
```

Comment:
I would love comments about what these various tuning parameters do, and their effect, but that's not blocking.

Reply:
They're eventually going to be controllable via Rust code. I just haven't exposed them to users yet. When I do, I'll add doc comments of course.